main.py

```diff
@@ -132,7 +135,32 @@ def handle_args():
 def main():
     config = handle_args()
     logger.setLevel(config.log)
-    AudiobookGenerator(config).run()
+
+    if os.path.isdir(config.input_file):
+        print("Searching for .epub files...")
+        epub_list = [y for x in os.walk(config.input_file) for y in glob(os.path.join(x[0], '*.epub'))]
+        print(f"Found {len(epub_list)} .epub files.")
+
+        input_folder_root = str(Path(config.input_file))
+        output_folder_base = config.output_folder
+
+        for epub_file in epub_list:
```

Comment:
No, sorry.
Maybe you would love to have a folder with a bunch of epubs and want to convert them all in one go, but no. please 1 book at a time, especially if you use OpenAI and each book costs money